app/models/user.rb

```diff
@@ -44,6 +44,7 @@ class User < ApplicationRecord
            source: :service_provider_record
   has_many :sign_in_restrictions, dependent: :destroy
   has_many :in_person_enrollments, dependent: :destroy
+  has_many :fraud_events # TBD, do we want to support multiple?
```

Comment:
Users aren't _meant_ to have multiple of these, and I'm not sure there's an obvious way. I'm tempted to make this a `has_one`. But am I overlooking some possible edge case?

I guess `dependent: :destroy` would be safe here.